arkouda/akstats/_stats_py.py

```diff
@@ -0,0 +1,212 @@
+from collections import namedtuple
+
+import numpy as np
+from numpy import asarray
+from scipy.stats import chi2
+
+import arkouda
```

Comment:
can we do `arkouda as ak`? Seeing `arkouda.array` in the below code just looks strange

Reply:
Sure.  It was pointed out in the meeting that it might be better in the future to "import arkouda as np" to integrate with the library better.